### ISSUE TITLE
Add button to switch from <-> to

### ISSUE THREAD

**Issue by xivk** [anyways-open/routing-api-test-frontend#9]
Add button to switch from <-> to

**Comment by xivk** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [More test frontend debugging features.](https://trello.com/c/TE9BkXHd/10-more-test-frontend-debugging-features)

**Comment by pietervdvn** [comment]
Additionally, a button to clear the departure location could be useful to - or even better, a way to quickly _set_ the departure location, e.g. middle mouse click to set it.

**Comment by xivk** [comment]
Good idea, also a third click on the map could be a good way to add a via point and the let users drag the markers
